src/github.com/getlantern/lantern-mobile/app/src/main/res/values/styles.xml

```diff
@@ -1,7 +1,7 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
```

Comment:
Nice! It looks like ``AppTheme`` is used by default whenever our main activity extends ``AppCompatActivity`` but only with 5.x -- anyway, just verified the black status bar is gone now! Merging.